prow/config.yaml

```diff
@@ -252,6 +254,8 @@ presubmits:
         - "--pull=$(PULL_REFS)"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
         - "--git-cache=/root/.cache/git"
+        - "--timeout=75"
+        - "--json"
```

Comment:
Oh yeah, good point. I'll add a personal TODO to follow this up with another PR.